### DISCUSSION TITLE
Update from EME 2.2.1

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#625] [Q&A] [answered]
I have built a website a few years ago. And now, it should go online.
An there is a very old version of EME installed. If I try to update this, I get an error.
How can I get this to the up to date version?
Or can I export the events / locations and import them again, if there is installed an actual version?

Thank you!

**Comment by liedekef** [comment, marked as answer]
2.2.1 is from June 2021 ... anyway, it should work but your php version should be up to date too. What is the error you're getting?

**Comment by ebussinetz** [comment]
OK, you are right, I have forgotten to change the PHP version.
Thank you!
